app/components/modals/invite-new-user.js

```diff
@@ -46,6 +47,12 @@ export default ModalComponent.extend(ValidationEngine, {
         this.set('hasValidated', emberA());
     },
 
+    // didRender() {
```

Comment:
So instead of doing a `run.later`, try this:

```js
import run, {scheduleOnce} from 'ember-runloop'; // at top of file

// in place of didRender
didInsertElement() {
    scheduleOnce('afterRender', () => {
        this.$('#new-user-email').focus();
    });
}
```